expression/expr_to_pb.go

```diff
@@ -266,20 +267,20 @@ func (pc PbConverter) scalarFuncToPBExpr(expr *ScalarFunction) *tipb.Expr {
 		children = append(children, pbArg)
 	}
 
-	var implicitArgs []byte
-	if args := expr.Function.implicitArgs(); len(args) > 0 {
-		encoded, err := codec.EncodeValue(pc.sc, nil, args...)
+	var metadata []byte
+	if args := expr.Function.metadata(); args != nil {
```

Comment:
name it metadata?

Reply:
No variable shadow in golang, metadata already exists in L270..

Reply:
encoded_metadata XD

Reply:
encodedMetadata

Reply:
use encoded